### PR TITLE
X1-AV-30-SendToBlackBox

### PR DESCRIPTION
**Pull request by macsyd**
A function that appends a list to a csv. Needs a name for the file it's saving to.

### CODE REVIEW

File: src/dataHandler.py [hunk 1]
```diff
@@ -63,6 +64,13 @@ def send_strain_gauges(self, sendingTo):
         return
     
     def send_all_data(self, sendingTo):
```
Comment (klemie) [inline src/dataHandler.py:66]:
```suggestion
    def send_all_data(self, sendingTo, destination):
```
Destination can = name of file so we dont need to hardcode it in

File: src/dataHandler.py [hunk 2]
```diff
@@ -63,6 +64,13 @@ def send_strain_gauges(self, sendingTo):
...
+        if(sendingTo == blackbox):
```
Comment (klemie) [inline src/dataHandler.py:67]:
```suggestion
        if sendingTo == 'blackbox':
```
python doesnt need `()` in if statements. I know its weird.

File: src/dataHandler.py [hunk 3]
```diff
@@ -63,6 +64,13 @@ def send_strain_gauges(self, sendingTo):
...
+            data = self.convert_to_csv()
+            f = open('theNameOfTheFile.csv', 'a')
```
Comment (klemie) [inline src/dataHandler.py:69]:
```suggestion
            f = open(destination, 'a')
```

File: src/dataHandler.py [hunk 4]
```diff
@@ -63,6 +64,13 @@ def send_strain_gauges(self, sendingTo):
...
+            writer = csv.writer(f)
+            writer.writerow(data)
+            f.close()
+        
         return
```
Comment (klemie) [inline src/dataHandler.py:74]:
```suggestion

```
no return needed

File: src/dataHandler.py [hunk 5]
```diff
@@ -1,4 +1,5 @@
 from rocketData import RocketData as rd
+import csv
 class SendData:
     """
     This only deals with sending the rocketdata. RocketData class in src/rocketData.py deals with manipulating the data.
```
Comment (klemie) [inline src/dataHandler.py:5]: can you edit the description of the method in this comment block to say it gets saved to a csv?

**Review by klemie — CHANGES_REQUESTED** [review]
Look good to me. Just tought of a quick fix to not knowing where to write to.

File: src/dataHandler.py (hunk 5 above)
Comment (macsyd) [reply src/dataHandler.py:5]: I have updated it

**Review by klemie — CHANGES_REQUESTED** [review]
Forgot one small thing. After this it'll be ready to merge

**Pull request merged**